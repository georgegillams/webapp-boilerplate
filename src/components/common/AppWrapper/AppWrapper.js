import React from 'react';

import { StyledAppWrapper } from './app-wrapper.styles';

const AppWrapper = props => {
  const { ...rest } = props;

  return <StyledAppWrapper {...rest} />;
};

AppWrapper.propTypes = {};

AppWrapper.defaultProps = {};

export default AppWrapper;
