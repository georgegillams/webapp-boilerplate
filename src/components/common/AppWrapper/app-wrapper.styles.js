import styled from 'styled-components';

export const StyledAppWrapper = styled.div`
  display: flex;
  min-height: 100vh;
  flex-direction: column;
`;
