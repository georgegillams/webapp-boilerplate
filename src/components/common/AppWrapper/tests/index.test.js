import React from 'react';

import { render } from '@testing-library/react';

import AppWrapper from '..';

describe('<AppWrapper />', () => {
  const children = <div>some text</div>;

  it('Should render correctly', () => {
    const { container } = render(<AppWrapper>{children}</AppWrapper>);

    expect(container).toMatchSnapshot();
  });
});
