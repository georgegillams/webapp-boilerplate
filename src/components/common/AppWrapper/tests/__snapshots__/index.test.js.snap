// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppWrapper /> Should render correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  min-height: 100vh;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div>
  <div
    class="c0"
  >
    <div>
      some text
    </div>
  </div>
</div>
`;
